examples/api/lib/material/chip/chip_action.0.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Good catch, this was all part of one class before, I was the one who split the chip classes into separate files.

Reply:
>  It's unfortunate that someone named the source file material/chip_action.dart, since it doesn't match the class name.

I named them this way so all chip classes are next to each other. 

If that's not a breaking change, I could rename these classes to match widget names.

Reply:
Also removed commented code that snuck in by  https://github.com/flutter/flutter/pull/107166/files#r915230787

Reply:
> If that's not a breaking change, I could rename these classes to match widget names.

Do you mean renaming the files?

Renaming the classes is definitely a breaking change: Chips used a lot in first party apps. Renaming the files is however, not a breaking change.

Reply:
Renamed the files :)

Reply:
Yes, then that would be great: it would make the naming more consistent.

Then the name of this file could be:
```
examples/api/lib/material/action_chip/action_chip.0.dart
```